modules/flake-parts/all-modules.nix

```diff
@@ -31,7 +31,13 @@ in {
   };
 
   # generates future flake outputs: `modules.<kind>.<module-name>`
-  config.flake.modules = lib.mapAttrs (kind: _: mapModules kind) moduleKinds;
+  config.flake.modules =
```

Comment:
I've assumed that this was for internal purposes, and I think it would make sense to have something like this in flake-parts for publishing modules.

It would also be nice to have an option for local modules that aren't published.